openpype/hosts/maya/api/plugin.py

```diff
@@ -262,21 +281,7 @@ def get_pre_create_attr_defs(self):
 
     def apply_settings(self, project_settings, system_settings):
         """Method called on initialization of plugin to apply settings."""
-
-        settings_name = self.settings_name
-        if settings_name is None:
-            settings_name = self.__class__.__name__
-
-        settings = project_settings["maya"]["create"]
-        settings = settings.get(settings_name)
-        if settings is None:
-            self.log.debug(
-                "No settings found for {}".format(self.__class__.__name__)
-            )
-            return
-
-        for key, value in settings.items():
-            setattr(self, key, value)
+        apply_settings(self, project_settings, system_settings)
```

Comment:
This class is missing `settings_name = None` set at the top.

Reply:
`MayaCreator` does have the attribute `settings_name = None`